lib/phobos_db_checkpoint/event_helper.rb

```diff
@@ -0,0 +1,15 @@
+module PhobosDBCheckpoint
+  module EventHelper
+    def configured_handler
+      Phobos
+        .config
+        .listeners
+        .find { |l| l.group_id == self.group_id }
+        .handler
+        .constantize
+    rescue NoMethodError => e
+      raise(HandlerNotFoundError, self.group_id) if e.message =~ /handler/
```

Comment:
if we split the calls into `listener = ...` then try `listener.handler.constantize` we can do the nil check and raise in between.
I prefer that over trying to see which method was not found.